service-api/src/main/java/greencity/dto/event/CoordinatesDto.java

```diff
@@ -11,6 +11,13 @@
 public class CoordinatesDto {
     private double latitude;
     private double longitude;
-    private String addressUa;
-    private String addressEn;
+    private String streetEn;
```

Comment:
It is not `CoordinatesDto` anymore
It contains other data also
Please rename it

Reply:
renamed